crates/driver/src/tests/cases/protocol_fees.rs

```diff
@@ -37,6 +41,17 @@ struct TestCase {
     order: Order,
     fee_policy: Policy,
     execution: Execution,
+    expected_score: eth::U256,
+}
+
+// because of rounding errors, it's good enough to check that the expected value
+// is within a very narrow range of the executed value
+fn is_approximately_equal(executed_value: eth::U256, expected_value: eth::U256) -> bool {
```

Comment:
This function is copy/pasted from `e2e tests` crate, maybe we can define it in a common trait :thinking:

Reply:
Probably, but I couldn't find a good place for it.